requirements.txt

```diff
@@ -13,5 +13,6 @@ networkx
 pathlib
 nose
 nose-exclude
+typing-extensions
 xlwt
 git+https://github.com/scikit-rf/scikit-rf.git@master
```

Comment:
please update with the master branch, the list of requirements has changed since #459 and #460